ember/app/controllers/dependencies/index.js

```diff
@@ -0,0 +1,28 @@
+import Controller from '@ember/controller';
+import { tracked } from '@glimmer/tracking';
+
+export default class DependencyIndexController extends Controller {
+  queryParams = ['name', 'provider'];
+
+  @tracked name = null;
+  @tracked provider;
+
+  @tracked model;
+
+  get providers() {
+    return ['PIP', 'NPM'];
+  }
```

Comment:
Could become a future improvement: make the available providers configurable or request this information from the backend. Hardcoding this in some buried controller is not the way to go. :-)

Reply:
I created an issue for it: https://github.com/adfinis/Outdated/issues/186